### PR TITLE
changes to css to make home page more responsive.

### LINKED ISSUE #115: [BUG]: The introduction section is not designed properly and also the mobileview of that is not good can be changed.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Actually the introduction section of the website in home page is not looking good with inappropriate margins and padding.
Also the mobile view is looking very bad and it can broken with the column flex direction.

### Expected Behavior

The desktop view should be maintained with proper margin and padding.
The mobile view should also be improved with proper direction of the sections and the image and text should be one below other.

### Steps To Reproduce

1.Open the project
2.Preview the home page(index.html)
3.See the middle(introduction) section where there is image and text
4.See the middle(introduction) section in mobile view

### Anything else?

image

### What browsers are you seeing the problem on?

_No response_

### PR DESCRIPTION
**Pull request by CampbellBrim**
## Motivation and Context

improve responsiveness of home page

Fixes #115 

## Description

Changes to make flex direction change on smaller screen sizes. Adjustments to margins. Font color changed to black

## How Has This Been Tested?

chrome dev tools responsive viewport mode

## Screenshots (if appropriate):
before:
Captura de tela 2024-03-27 134259
After: 
Captura de tela 2024-03-27 134226

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -90,26 +90,25 @@ nav ul li a:active {
...
   border-radius: 4% 4% 25% 4%;
-  max-width: 80%;
+  width: 100%;
   height: auto;
   position: relative;
 }
 
 .intro-text {
-  color: #fff;
+  color: black;
```
Comment (adityaraute) [inline assets/css/style.css:107]: Before:
image


After:
image

The text is too wide. I liked the previous margin settings, could you revert this change?

Also, hadn't we discussed we'd choose a shade different than black or white? Perhaps we can try a lighter shade here.

File: assets/css/style.css [hunk 2]
```diff
@@ -373,6 +371,10 @@ nav ul li a:active {
 }
 
 @media (max-width: 950px) {
+  .introduction {
```
Comment (adityaraute) [inline assets/css/style.css:374]: Even in the mobile view
image

I'm not happy about how close the text is to the margins and the image above. Perhaps we could add some space around the text here?

**Review by adityaraute — CHANGES_REQUESTED** [review]
Apart from the stylistic changes suggested, I feel there are a lot of unnecessary changes in this PR to the style.css file. 
I would appreciate it if you could get rid of those changes and make the PR crisper.
Thanks!

**Comment by CampbellBrim** [comment]
> Apart from the stylistic changes suggested, I feel there are a lot of unnecessary changes in this PR to the style.css file. I would appreciate it if you could get rid of those changes and make the PR crisper. Thanks!

When I reduced the margins it created side scrolling. I believe this was because of the header. I made multiple changes to try to remove the side scrolling, but was having a hard time understanding why there was side scrolling. I will reduce the changes.

**Pull request closed without merging**